{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -37,7 +37,7 @@ deploy:
   password:
     secure: PLEASE_REPLACE_ME
   on:
-    branch: master
+    branch: stable
```

Comment:
Maybe a commented pointer here to https://help.github.com/en/articles/configuring-protected-branches
Something like:
```
# To limit who can create releases, see how to set branch protection here:
# https://help.github.com/en/articles/configuring-protected-branches 
```